### PR TITLE
Fix undefined variables in installer

### PR DESCRIPTION
**Pull request by rkiyanchuk**
Fix few undefined variables found by static check in installer code.

I replaced undefined `log_path` with `storage['LOG_PATH']`, and further at `genfstab` step there was undefined `o` varaible in error log message: not immediately clear what it's supposed to reference, so I just removed it for now. Please let me know if the intent was to log some other variable there!

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code to the best of my abilities
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where possible/if applicable

### CODE REVIEW

**Review by Torxed — CHANGES_REQUESTED** [review]
Looks great, if it's not to much trouble push a commit to add the debug output back which I forgot to correct way back when :)

File: archinstall/lib/installer.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def genfstab(self, flags='-pU'):
 			fstab_fh.write(fstab)
 
 		if not os.path.isfile(f'{self.mountpoint}/etc/fstab'):
-			raise RequirementError(f'Could not generate fstab, strapping in packages most likely failed (disk out of space?)\n{o}')
+			raise RequirementError(f'Could not generate fstab, strapping in packages most likely failed (disk out of space?)\n')
```
[the remark this answers is not in the document]
Comment (Torxed) [reply archinstall/lib/installer.py]: I went ahead and added the log back :)

**Review by Torxed — APPROVED** [review]

**Comment by rkiyanchuk** [comment]
My bad for the delay, was out traveling last week :) Thanks for merging anyway!

**Comment by Torxed** [comment]
No worries, I realized I can commit fixes to your branch before merging hehe. So no worries :)

**Pull request merged**